docker-compose.yml

```diff
@@ -60,7 +60,7 @@ services:
       - TJENESTEBUSS_INTEGRASJON_SCOPE=api://dev-fss.pensjonsbrev.pensjonsbrev-tjenestebuss-lokal/.default
       - NAVANSATT_URL=https://navansatt.dev-fss-pub.nais.io
       - NAVANSATT_SCOPE=api://dev-fss.teampensjon.navansatt/.default
-      - GROUPS_PENSJON_UTLAND=bda6bd68-77e6-4c00-96b9-7ecf5df7413c
+      - GROUPS_PENSJON_UTLAND=dec3ee50-b683-4644-9507-520e8f054ac2 #TODO, to make dev work
```

Comment:
Endret til gruppa vi har i dev for å få opp eblanketter...Burde kanskje gjøre noe smartere her på sikt?